buildpacks/nodejs-engine/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update default node version to 20.x
```

Comment:
(To keep tenses consistent with both prior entries, plus the `### Changed` headings etc)

```suggestion
- Updated default node version to 20.x
```